### PR TITLE
New volunteer page

### PR DESCRIPTION
**Pull request by symroe**

### CODE REVIEW

File: ynr/templates/volunteer.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+We need your help collecting candidate information. Read this guide to understand how you can help!
+
+Democracy Club Candidates is an open-source database of all candidates standing in UK elections. Its data powers
+Democracy Club’s [Who Can I Vote For?](https://whocanivotefor.co.uk/), and is regularly used by a wide range of
+organisations, including [The Electoral Commission](https://www.electoralcommission.org.uk/), social media
+companies, and national newspapers.
+
+## How to help crowdsource candidates
+
```
Comment (VirginiaDooley) [inline ynr/templates/volunteer.html:16]: "You can help crowdsource candidates"...reframing as "You" feels more empowering.

File: ynr/templates/volunteer.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Elections in the UK are run on a local level by individual councils. This means that the state does not collect a
+central list of candidates in a given election. This site aims to fill that gap. Hundreds of brilliant volunteers,
+like you, come together to make this data available.
+
+First you must [create your account](https://candidates.democracyclub.org.uk/accounts/signup/).
+
```
Comment (VirginiaDooley) [inline ynr/templates/volunteer.html:22]: Maybe just add something about why we require accounts (aka another thing to sign up for)?

File: ynr/templates/volunteer.html [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+WhoCanIVoteFor, depending on current site traffic.
+
+## Adding candidates to an election
+
+You can add candidates to forthcoming elections on the candidate’s profile, or add an existing or new candidate
+directly to the relevant ballot, selected from the ballot column on the
+[list of upcoming elections](https://candidates.democracyclub.org.uk/elections/). Alternatively, you can ‘bulk add’
+all candidates for a single party by using the “add candidates for a single party” feature, accessed by selecting the
+relevant election from the election column.
+
```
Comment (VirginiaDooley) [inline ynr/templates/volunteer.html:37]: Screenshots might help, but it's clear in any case.

File: ynr/templates/volunteer.html [hunk 4]
```diff
@@ -0,0 +1,153 @@
+{% extends "base.html" %}
+{% load markdown_deux_tags %}
+{% block content %}
+  {% filter markdown%}
+<div markdown="1">
+# How to Volunteer: collect candidate information
+
+We need your help collecting candidate information. Read this guide to understand how you can help!
+
```
[the remark this answers is not in the document]
Comment (pmk01) [reply ynr/templates/volunteer.html:9]: No, we've used it before, I think it's a good change.

**Review by michaeljcollinsuk — APPROVED** [review]

**Comment by symroe** [comment]
@VirginiaDooley thanks for the suggestions! Can you work with @pmk01 to make the changes and merge when you're both ok with it?

I think the vibe here is "something is better than nothing" rather than aiming for perfection :+1:

File: ynr/templates/volunteer.html (hunk 2 above)
Comment (VirginiaDooley) [reply ynr/templates/volunteer.html:22]: @pmk01 Could you provide exact text for this and I'll add?

Comment (pmk01) [reply ynr/templates/volunteer.html:22]: User accounts are necessary for transparency and security of the database - all edits are logged and recorded on each candidate profile. Edits are reviewed by Democracy Club staff, and users will be banned if vandalism is suspected.

Comment (pmk01) [reply ynr/templates/volunteer.html:22]: howzat? thoughts welcome!

**Pull request merged**